pages/_app.js

```diff
@@ -34,6 +34,10 @@ class Application extends App {
             href="https://fonts.googleapis.com/css?family=Lato:400,700|Roboto:300,400,700&display=swap"
             rel="stylesheet"
           />
+          <link
+            rel="gitview icon"
+            href="./static/favicon.ico"
+          />
```

Comment:
We should have all the favicons here from the generated markup like the following from https://realfavicongenerator.net/

```
<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
<link rel="manifest" href="/site.webmanifest">
<link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
<meta name="msapplication-TileColor" content="#ffffff">
<meta name="theme-color" content="#ffffff">
```

This will better ensure SEO